### DISCUSSION TITLE
Restore firmware from USB-C

### DISCUSSION THREAD

**Discussion by negronoir** [serkri/SmartEVSE-3#218] [Q&A]
Hello everybody... I've briked one of my two SmartEVSE 

I've followed thw instructions on main page 'Building the firmware' under Windows 10.

I've modified some source code of 1.7.3

Minor changes... Only

#define MIN_CURRENT 6 -> #define MIN_CURRENT 5 

and

#define MQTT 1 -> #define MQTT 0

I've compiled apparently right and uploaded new firmware.bin through webserver, I've received OK message but, after that... Blank screen. No text, no AP, no webserver, no reaction to buttons and always backlight is on.

I've tried to restart the SmartEVSE switching the circuit breaker thinking in some sort of rollback mechanism but no luck.

So I think that the only way to revive device is using USB-C

I'm not a very pro programmer so... Somebody could please tell me a clear guidance to restore 1.7.3 'stock' release on the device into a Windows environment? I suppose that is not as easy as put firmware.bin & spiffs.bin into a USB stick root folder and power on the device... Probably some sort of modem COM connection shall be done through this port.

Thanks for any help!!

Antonio

**Comment by dingo35** [comment]
Look here:
https://github.com/serkri/SmartEVSE-3/discussions/217

**Reply by negronoir** [reply]
Like a charm! Thanks. Works again.

P.S. Apologise but I dind't change "open questions" flag and I didn't see #217 question
